server/apiCalls/ladokApi.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const { createApiClient } = require('om-kursen-ladok-client')
+const serverConfig = require('../configuration').server
+
+async function getLadokCourseData(courseCode) {
+  const client = createApiClient(serverConfig.ladokMellanlagerApi)
+  const course = await client.getLatestCourseVersion(courseCode)
+  return course
+}
+
+const filteredLadokData = async (courseCode, lang) => {
+  const { apiError, huvudomraden, benamning, kod, omfattning, statusCode } = await getLadokCourseData(courseCode)
+  return {
+    apiError,
+    mainSubject: lang == 'sv' ? huvudomraden[0].sv : huvudomraden[0].en,
+    courseTitleData: {
+      courseCode: kod,
+      courseTitle: lang == 'sv' ? benamning.sv : benamning.en,
```

Comment:
I've made some changes in `om-kursen-ladok-client` so benamning is a string and now a {sv, en} object anymore. Just pass lang to `getLatestCourseVersion()` instead and use just `benamning` here.


```suggestion
      courseTitle: benamning,
```

Reply:
It was changed here https://github.com/KTH/studadm-om-kursen-packages/pull/1